git_machete/client.py

```diff
@@ -1852,12 +1852,12 @@ def checkout_github_prs(self,
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
         if not current_user and my_opened_prs:
+            msg = f"Could not determine current user name, please check your {GITHUB_TOKEN_ENV_VAR} environment variable."
```

Comment:
Hmm the problem is that well, `GITHUB_TOKEN` is not the only way of providing the token... in fact this message might be confusing if someone relied on `gh`/`hub` authentication instead (and the token got expired or whatnot). Maybe just display the full(er) message as in github_api_access.rst ?

Reply:
Or just redirect the user to `git machete help github` for details on how to provide the required auth!

Reply:
Actually! see `git_machete.github.__fire_github_api_request`. Would be good to have a common handler code for the problems with GitHub token (the ones in `__fire_github_api_request` are slightly out of date already, `~/.github-token` are not taken into account :/)